source/software/matlab_parallel_computing.rst

```diff
@@ -3,12 +3,155 @@
+   $ module use /apps/leuven/skylake/2021a/modules/all
+   $ module load MATLAB/R2022b #MATLAB/R2022a if you would like to load this version
+    
+Once the module is loaded, start Matlab. You can start it in a terminal without the GUI by using the ``-nodisplay`` flag. Then, run the ``configCluster`` script.
+This will set up the parallel configuration.  
+
+::
+
+   $ matlab -nodisplay
+
+::
+
+   >> rehash toolbox;
+   >> configCluster;
+   
+You will get a message as follows if it succeeded: 'Complete.  Default cluster profile set to "<cluster_name> <matlab_version>".'. You have now created a cluster
+profile.
+
+Configuring additional properties
+*********************************
+
+It is possible to set some user-specific properties and save them in your cluster profile. You can always adapt these and save the profile again, without performing
+the previous step again. All of these properties are available in the ``AdditionalProperties´´ property of your cluster profile. 
+   
+Before you will be able to adapt these properties, you first need to get a handle on the cluster:
+
+::
+
+   >> c = parcluster;
+   
+``c = parcluster´´ will automatically load the correct profile based on the Matlab module you have loaded. So if you would have a profile for both R2022a and R2022b,
+the correct profile will be chosen when using this command.
+
+If you would prefer to use the local resources (i.e. the resources of the node you are currently on), use:
+
+::
+
+   >> c = parcluster('local');
+
+You can set a range of additional properties on each profile. There are two properties that are required, namely ``AccountName`` and ``WallTime``. Set any 
+other to your preference. Once these are saved, they will be kept in the settings of your cluster profile, but they can always be adapted again later.
+
+::
+
+   >> % View the additional properties
+   >> c.AdditionalProperties
+   >> % set the required properties
+   >> c.AdditionalProperties.AccountName = <account-name>;
+   >> c.AdditionalProperties.WallTime = '05:00:00';
+   >> % save the profile
+   >> c.saveProfile
+   
+Now the profile is ready, and you should be able to submit multi-node jobs with Matlab now.
+
+Submitting jobs
++++++++++++++++
+
+Submitting jobs will happen from within Matlab. First of all, you will need to start an interactive session on the cluster. It is possible to use the GUI, but then
+you will need to request GPU resources. You can also open the GUI on NX, but you need to submit your jobs from a compute node. Submitting from a login node
+will cause an error. 
+
+Once you have an active interactive job, load the module and start Matlab. Now you can start both interactive and independent batch jobs with the previously configured 
+cluster profile. Follow the steps below to submit a job.
+
+Interactive job
+***************
+
+You can start an interactive job using the ``parpool`` function:
+
+::
+
+    >> c = parcluster;
+    >> p = parpool(64); % requesting 64 cores
```

Comment:
Could you just add one or two more lines about what is supposed to be happening now? I.e.:
* When the `parpool` call returns, does that mean that your interactive job has started to run?
* If you then want to execute something in that interactive job (and not in the parent interactive
  Matlab prompt), what do you then need to do?

(The batch job example is fairly clear about this, but for the interactive case this can be made clearer.)